testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -1534,6 +1534,8 @@
                                     <auth.server.migration>true</auth.server.migration>
                                     <keycloak.migration.home>${containers.home}/auth-server-migration</keycloak.migration.home>
                                     <migration.import.props.previous>${migration.import.props.previous}</migration.import.props.previous>
+                                    <auth.server.feature>${auth.server.feature}</auth.server.feature>
```

Comment:
Do you mean for this key to refer to itself, and be the same as the key on the next line?

Reply:
Actually, only the next line is needed. This one was an attempt to make it configurable but for migration we just want to enable it by default.

The value from the last one is what we need and the value actually set.

Will send a PR fixing this ... Tks.